src/clj_puppetdb/vcr.clj

```diff
@@ -22,21 +21,29 @@
         (assoc response :content-type
                         {:mime-type (.getMimeType content-type)
                          :charset   (.getCharset content-type)})))))
+
 (defn body->string
   [response]
   "Turn response body into a string."
   (let [charset (util/response-charset response)]
-    (update-in response [:body] #(slurp (BufferedReader. (InputStreamReader. % charset))))))
+    (->> (-> (get response :body)
```

Comment:
Just a note: We can't use
```clojure
(slurp % :ecoding charset)
```
as ```charset``` is an instance of ```java.nio.charset.Charset``` while the argument to ```:encoding``` must be a string.